src/test/kotlin/com/autonomousapps/internal/advice/ConsoleReportTest.kt

```diff
@@ -0,0 +1,305 @@
+package com.autonomousapps.internal.advice
+
+import com.autonomousapps.Ignore
+import com.autonomousapps.advice.Advice
+import com.autonomousapps.advice.Dependency
+import com.autonomousapps.internal.Component
+import org.junit.Test
+import kotlin.test.*
+
+class ConsoleReportTest {
+
+  private val orgDotSomethingV1 = Dependency("org.something", "1.0")
+  private val orgDotSomethingV2 = Dependency("org.something", "2.0")
+  private val componentV1 = Component(
+    dependency = orgDotSomethingV1,
+    isTransitive = false,
+    isCompileOnlyAnnotations = true,
+    classes = emptySet()
+  )
+  /* ****************************************
+   * Change advice (incorrect configurations)
+   * ****************************************
+   */
+
+  @Test fun `isEmpty should return true when empty`() {
+    // When
+    var consoleReport = createReport()
```

Comment:
all of these `var`s can, and should, be `val`s. Always default to `val`. Kotlin Rule #1. Only use `var` if you can't, for some reason, use `val`.